queue/abstract.lua

```diff
@@ -613,6 +685,38 @@ local function build_stats(space)
     return stats
 end
 
+--- Identificate the session.
+-- If "queue_uuid" == nil - create a new UUID for the session
+-- and return it, otherwise connect to the existing one.
+function method.identificate(queue_uuid)
+    local queue_session_ids = box.space._queue_session_ids
+    local sid = session.id()
+    local session_ids = queue_session_ids.index.box_sid:get(sid)
+    local quuid = session_ids and session_ids[2]
+
+    if queue_uuid == nil and quuid == nil then
```

Comment:
Can we name those UUIDs somehow like `new_uuid` and `old_uuid`? It is hard to me to understand what is what when I look into some hunk within the function: the `queue_uuid` looks quite same to the `quuid`.

Reply:
Done.